.pre-commit-hooks.yaml

```
@@ -15,7 +15,7 @@
   pass_filenames: false
 - id: kubepug-minimum
   name: KubePug - k8s v1.15 minimum
-  entry: bash -c 'find . -name Chart.yaml | xargs -n1 -I% bash -c "dirname %" | xargs -t -n1 -I% bash -c "helm template % --kube-version 1.15 | kubepug --error-on-deprecated --error-on-deleted --k8s-version v1.18.0 --input-file=-"'
+  entry: bash -c 'find . -name Chart.yaml | xargs -n1 -I% bash -c "dirname %" | xargs -t -n1 -I% bash -c "helm template % --kube-version 1.15 | kubepug --error-on-deprecated --error-on-deleted --k8s-version v1.18.1 --input-file=-"'
```

Is this really a change we want to do on releases?

---

oh gosh no, we need to improve the regexp of the release.sh script

---

```suggestion
  entry: bash -c 'find . -name Chart.yaml | xargs -n1 -I% bash -c "dirname %" | xargs -t -n1 -I% bash -c "helm template % --kube-version 1.15 | kubepug --error-on-deprecated --error-on-deleted --k8s-version v1.15.0 --input-file=-"'
```
It looks like the first time it was changed unwantedly was with 2a9a5b2d7042dc2d66a2aff5d96ca3eb2d973672